packages/hardware-trezor/src/transformers/withdrawals.ts

```diff
@@ -0,0 +1,42 @@
+import * as Trezor from 'trezor-connect';
+import { Cardano } from '@cardano-sdk/core';
+import { InvalidArgumentError } from '@cardano-sdk/util';
+import { TrezorTxTransformerContext } from '../types';
+import { resolveStakeKeyPath } from './keyPaths';
+
+export const toWithdrawal = (
```

Comment:
Let's rename this to `toTrezorWithdrawal` to make it clear what it transforms to

Reply:
@DominikGuzei Since we are already in Trezor package do we need to prefix functions?

Reply:
@DominikGuzei ? 👆

Reply:
@tomislavhoracek hm, yeah you are right 👍 let's keep it